src/vis/sidebar/VisTypeSelect.tsx

```diff
@@ -9,21 +9,18 @@ interface VisTypeSelectProps {
 
 export function VisTypeSelect({ callback, currentSelected }: VisTypeSelectProps) {
   return (
-    <>
-      <label className="pt-2 pb-1">Visualization Type</label>
-      <Select
-        closeMenuOnSelect
-        // components={{Option: optionLayout}}
-        onChange={(e) => callback(e.value)}
-        name="visTypes"
-        options={allVisTypes.map((t) => {
-          return {
-            value: t,
-            label: t,
-          };
-        })}
-        value={{ value: currentSelected, label: currentSelected }}
-      />
-    </>
+    <Select
+      label="Visualization type"
+      // components={{Option: optionLayout}}
+      onChange={(e) => callback(e as ESupportedPlotlyVis)}
+      name="visTypes"
+      data={allVisTypes.map((t) => {
+        return {
+          value: t,
+          label: t,
+        };
+      })}
```

Comment:
Switch visualization type to sentence case: 

![grafik](https://user-images.githubusercontent.com/5851088/195290200-1f80fe7f-4d5a-4333-8300-b5f3795a4f21.png)

Reply:
done